src/main/java/org/sagebionetworks/web/client/place/users/RegisterAccount.java

```diff
@@ -1,19 +1,17 @@
 package org.sagebionetworks.web.client.place.users;
 
-import com.google.gwt.place.shared.Place;
 import com.google.gwt.place.shared.PlaceTokenizer;
 import com.google.gwt.place.shared.Prefix;
+import org.sagebionetworks.web.client.place.ParameterizedPlace;
 
-public class RegisterAccount extends Place {
+public class RegisterAccount extends ParameterizedPlace {
```

Comment:
Change to a parameterized place so we can pass both the email and the signed token to the register account page (which will be passed to One Sage)